zap/src/test/java/org/parosproxy/paros/network/HttpRequestHeaderUnitTest.java

```diff
@@ -204,6 +204,17 @@ void shouldReplaceAnyCookieHeaderWhenSettingCookieParams() {
         assertThat(header.getHeaderValues(HttpHeader.COOKIE), hasSize(1));
     }
 
+    @Test
+    void shouldNotHaveContentLengthHeaderIfGetRequest() throws Exception {
```

Comment:
`shouldNotHaveContentLengthHeaderByDefault`, it's the same behaviour regardless of the method.

Reply:
done